jupyterlab/commands.py

```diff
@@ -38,6 +38,72 @@
 DEV_DIR = osp.abspath(os.path.join(HERE, '..', 'dev_mode'))
 
 
+class ProgressProcess(Process):
+
+    def __init__(self, cmd, logger=None, cwd=None, kill_event=None,
+                 env=None):
+        """Start a subprocess that can be run asynchronously.
+
+        Parameters
+        ----------
+        cmd: list
+            The command to run.
+        logger: :class:`~logger.Logger`, optional
+            The logger instance.
+        cwd: string, optional
+            The cwd of the process.
+        env: dict, optional
+            The environment for the process.
+        kill_event: :class:`~threading.Event`, optional
+            An event used to kill the process operation.
+        quiet: bool, optional
```

Comment:
`env` and `kill_event` are swapped in the docstring from the method signature, and `quiet` is not a parameter